modules/ding_webtrekk/ding_webtrekk.install

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ * Handle module installation and updates.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_webtrekk_install() {
+  _ding_webtrekk_set_cookie_compliance_text();
+}
+
+/**
+ * Update cookie compliance text with Webtrekk specific information.
+ */
+function ding_webtrekk_update_7001() {
+  _ding_webtrekk_set_cookie_compliance_text();
+}
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function ding_webtrekk_update_dependencies() {
+  // We need ding2 to add the general cookie information before we append the
+  // Webtrekk specific information.
+  $dependencies['ding_webtrekk'][7001] = array(
+    'ding2' => 7052,
+  );
+  return $dependencies;
+}
+
+/**
+ * Helper function to append the Webtrekk specific cookie compliance text.
+ */
+function _ding_webtrekk_set_cookie_compliance_text() {
+  $body = '<p><strong>Webtrekk</strong></p>
+    <p>Vi bruger Webtrekk til at føre statistik over trafikken på hjemmesiden. Al indsamlet statistik er anonym.<br>(<a class="external" href="https://www.webtrekk.com/en/legal/opt-out-webtrekk/">Webtrekk - om brug af cookies på websider og øvrige forhold omkring persondata</a>)<br>- Hvis du vil fravælge cookies fra Webtrekk kan du trykke her:
+    <p><a class="button webtrekk-opt-out" href="#">Fravælg Webtrekk cookie</a></p>
```

Comment:
This is fragile. We are tying key functionality to a special class set in a WYSIWYG editor text field. In other words: If an editor deletes this by accident it is hard to get back.

Reply:
Yes, this is unfortunate. Maybe a node isn't the right thing for this? I'll see if I can come up with a better solution.

Reply:
Updated PR: Both webtrekk opt-out text and javascript is now added in hook_node_view().